source/tutorials/importing.rst

```diff
@@ -44,8 +44,45 @@ Importing data
 
    qiime tools import \
      --type EMPSingleEndSequences \
-     --input-path raw-sequences \
-     --output-path raw-sequences.qza
+     --input-path emp-single-end-sequences \
+     --output-path emp-single-end-sequences.qza
+
+ "EMP protocol" multiplexed paired-end fastq
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ Format description
+ ******************
+
+ In the "Earth Microbiome Project (EMP) protocol" format for paired-end reads, there are three ``fastq.gz`` files, one containing forward sequence reads, one containing reverse sequence reads, and one containing the associated barcode reads, with the sequence data still multiplexed. The order of the records in the three ``fastq.gz`` files defines the association between the sequences reads and barcode reads.
+
+ Obtaining example data
+ **********************
+
+ .. command-block::
+
+    mkdir emp-paired-end-sequences
+
+ .. download::
+    :url: https://data.qiime2.org/2017.02/tutorials/atacama/emp-paired-end-sequences-1p/forward.fastq.gz
+    :saveas: emp-paired-end-sequences/forward.fastq.gz
+
+.. download::
+    :url: https://data.qiime2.org/2017.02/tutorials/atacama/emp-paired-end-sequences-1p/reverse.fastq.gz
```

Comment:
These two directive options must be aligned with the "d" in "download". As-is, Sphinx is interpreting those options as directive content (this is causing the Travis failure).